.github/workflows/delete-migrated-issue.yml

```diff
@@ -7,18 +7,17 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
     inputs:
-      name:
+      pr_number:
```

Comment:
Note that we need to update the other PR to match this.

Reply:
it's more like "updating this PR to match the other one" 😄 the other one already sends `pr_number`, this one was outdated